lib/facility_factory.rb

```diff
@@ -0,0 +1,46 @@
+class FacilityFactory
+  def initialize
+  end
+
+  def create_facility(api)
+    api.map do |facility|
+      Facility.new({
+        name: format_name(facility),
+        address: format_address(facility),
+        phone: format_phone_number(facility) 
+      })
+    end
```

Comment:
Great job using this map here! I do like that you used helper methods. A step further, we could have put all these new facility objects into an instance variable of `@facilities =[]` in the initialize method, that way we only have to do this mapping once. 

I read your message that you didn't test the helper methods, but we definitely would want to unit test them, to make sure that the input and output are consistent and are what we expect. Testing methods also allows us to test for sad path and edge cases to ensure the code doesn't break if an extra comma gets in or something.